services/motion/builtin/move_request.go

```diff
@@ -520,6 +520,7 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 	mr.seedPlan = seedPlan
 	mr.replanCostFactor = valExtra.replanCostFactor
 	mr.requestType = requestTypeMoveOnGlobe
+	mr.poseOrigin = nil // Needs to be unset
```

Comment:
Rather than needing to unset this for moveOnGlobe, could we please check if `relativeMoveRequestFromAbsolute` is doing moveOnMap or moveOnGlobe & have it only set `poseOrigin` if it is doing moveOnMap?

That way `relativeMoveRequestFromAbsolute` doesn't return a struct that is invalid for moveOnGlobe which the caller then needs to fix.

Reply:
Done